tests/Controller/CustomerApiTest.php

```diff
@@ -150,19 +150,20 @@ public function it_allows_to_get_customers_list()
         $this->assertResponse($response, 'customer/index_response', Response::HTTP_OK);
     }
 
-    /**
-     * @test
-     */
-    public function it_returns_page_not_found_if_limit_is_set_to_0()
-    {
-        $this->loadFixturesFromFile('authentication/api_administrator.yml');
-        $this->loadFixturesFromFile('resources/customers.yml');
-
-        $this->client->request('GET', '/api/v1/customers/?limit=0', [], [], static::$authorizedHeaderWithAccept);
-
-        $response = $this->client->getResponse();
-        $this->assertResponse($response, 'customer/page_not_found_response', Response::HTTP_NOT_FOUND);
-    }
+//    temporary disabled
```

Comment:
From our conversation, I know why it's easier and faster right now to disable it. However, I feel it needs a vast explanation, as it's not something we're usually doing in Sylius, so it's better to be transparent :)